packages/playground/src/components/pull-request/utils.tsx

```diff
@@ -93,3 +93,19 @@ export const timeDistance = (date1 = 0, date2 = 0, onlyHighestDenomination = fal
     minutes ? minutes + 'm' : hours || days ? '0m' : ''
   } ${seconds}s`
 }
+const LOCALE = Intl.NumberFormat().resolvedOptions?.().locale || 'en-US'
```

Comment:
Shouldn't this be same throughout the app everywhere? Not required right away but this should be set at App level itself so that it's available to everything below it.

Reply:
It should be the same throughout, I don’t think anywhere else has it set tho @vardanbansal-harness